src/main/java/org/openstreetmap/atlas/tags/filters/TaggableFilter.java

```diff
@@ -80,6 +80,17 @@ public static TaggableFilter forDefinition(final String definition)
         return new LineFilterConverter().convert(definition);
     }
 
+    /**
+     * @param definition
+     *            The {@link String} definition of the filter
+     * @deprecated Use {@code TaggableFilter.forDefinition(definition)} instead.
+     */
+    @Deprecated()
+    public TaggableFilter(final String definition)
+    {
+        this(TaggableFilter.forDefinition(definition));
```

Comment:
in previous behavior if definition is "", it'll return an instance that accepts everything.  with TaggableFilter.forDefinition(""), it will throw exception `org.openstreetmap.atlas.exception.CoreException: Taggable filter definition "" is invalid.`

Reply:
Fixed.